src/admin/components/elements/DatePicker/formattedLocales.ts

```diff
@@ -0,0 +1,25 @@
+// Locales must match the locales from date-fns
+// https://github.com/date-fns/date-fns/blob/main/src/locale/
+
+export const formattedLocales = {
```

Comment:
Can we change this so we import from src/translations/index, use Object.keys(translations), then overwrite the few that need to be directly changed?

This will make it one place to change the file when a new translation gets added where the datepicker already supports the same language code. There is an acceptable (in my mind) risk with this in that new langauges will need to be tested or checked against the supported languages of tnhe datepicker.